.github/workflows/e2e_tests.yml

```diff
@@ -52,6 +54,7 @@ jobs:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
           COMMIT_INFO_SHA: ${{ github.event.pull_request.head.sha }}
+          CYPRESS_AGORIC_NET: "${{ github.event_name == 'schedule' && 'emerynet' || 'local' }}"
```

Comment:
similarly, please factor out this expression. logic like this should be conspicuous to the reader

Reply:
can you further clarify what you mean by factoring out the expression? 
Since this is the recommended way to set the value of an env variable based on a condition, I don't think there's a simpler way of doing this
https://docs.github.com/en/actions/learn-github-actions/expressions#example

Reply:
I had in mind a job that produced these conditional values: https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs

But it would suffice to have a comment above these two ENV expressions calling out that they're conditional. I'd also put them at the top of the list. As someone is reading down the list they're likely to assume after the 10th one consistently being logic-less that the whole list is.